docker/dev.sh

```diff
@@ -1,8 +1,24 @@
-#!/bin/sh
+#!/bin/bash -e
+SCRIPTDIR=$(python -c 'import os,sys;print os.path.dirname(os.path.realpath(sys.argv[1]))' "$0")
```

Comment:
You do know that this trick is copied verbatim from the scripts you have made ? :)